internal/btpcli/facade_services_instance.go

```diff
@@ -72,7 +72,21 @@ func (f servicesInstanceFacade) Create(ctx context.Context, args *ServiceInstanc
 		return servicemanager.ServiceInstanceResponseObject{}, CommandResponse{}, err
 	}
 
-	return doExecute[servicemanager.ServiceInstanceResponseObject](f.cliClient, ctx, NewCreateRequest(f.getCommand(), params))
+	serviceInstanceResponseObject, cmdRes, err := doExecute[servicemanager.ServiceInstanceResponseObject](f.cliClient, ctx, NewCreateRequest(f.getCommand(), params))
+
+	//Workaround for NGPBUG-350117 => fix not feasible, keeping workaround
+	if cmdRes.StatusCode != 202 && err == nil {
+		return serviceInstanceResponseObject, cmdRes, err
+	} else if cmdRes.StatusCode == 202 && err == nil {
+		return f.GetByName(ctx, args.Subaccount, args.Name)
+	} else if err != nil {
```

Comment:
at this point, we always have err != nil

Reply:
True. Changed this to ELSE